_dashboards/management/scheduled-query-acceleration.md

```diff
@@ -0,0 +1,261 @@
+
+After this step, you can [run accelerated queries](#running-an-accelerated-query) using one of the secondary data sources. 
+
+## Running an accelerated query
+
+You can run accelerated queries in [Query Workbench]({{site.url}}{{site.baseurl}}/dashboards/query-workbench/). To run an accelerated query, use the following syntax:
+
+```sql
+CREATE SKIPPING INDEX example_index
+WITH (
+    auto_refresh = true,
+    refresh_interval = '15 minutes',
+    scheduler_mode = 'external'
+);
+```
+{% include copy.html %}
+
+## Parameters
+
+When creating indexes using an accelerated query, you can specify the following parameters in the `WITH` clause to control the refresh behavior, scheduling, and timing.
+
+| Parameter  | Description  | 
+|:--- | :--- | 
+| `auto_refresh`      | Enables automatic refresh for the index. If `true`, the index refreshes automatically at the specified interval. If `false`, refresh must be triggered manually using the `REFRESH` statement. Default is `false`.   |
+| `refresh_interval`  | Defines the time interval between refresh operations for the index, which determines how frequently new data is integrated into the index. This is applicable only when `auto_refresh` is enabled. The interval determines how frequently new data is integrated and can be specified in formats like `1 minute` or `10 seconds`. For valid time units, see [Time units](#time-units).| 
+| `scheduler_mode`    | Specifies the scheduling mode for auto-refresh (internal or external scheduling). The external scheduler requires a `checkpoint_location` (a path for refresh job checkpoints) for state management. For more information, see [Starting streaming queries](https://spark.apache.org/docs/3.5.1/structured-streaming-programming-guide.html#starting-streaming-queries). Valid values are `internal` and `external`.| 
+
+For more information and additional available parameters, see [Flint index refresh](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md#flint-index-refresh).
+
+## Time units
+
+You can specify the following time units when defining time intervals:
+
+- Milliseconds: `ms`, `millisecond`, or `milliseconds`
+- Seconds: `s`, `second`, or `seconds`
+- Minutes: `m`, `minute`, or `minutes`
+- Hours: `h`, `hour`, or `hours`
+- Days: `d`, `day`, or `days`
+
+## Creating a scheduled refresh job
+
+To create an index with a scheduled refresh job, use the following statement:
+
+```sql
+CREATE SKIPPING INDEX example_index
+WITH (
+    auto_refresh = true,
+    refresh_interval = '15 minutes',
+    scheduler_mode = 'external'
+);
+```
+{% include copy.html %}
+
+## Modifying refresh settings
+
+To modify refresh settings, use the `ALTER` command:
+
+```sql
+ALTER INDEX example_index
+WITH (refresh_interval = '30 minutes');
+```
+{% include copy.html %}
+
```

Comment:
there are some overlaps between ` ## Managing scheduled jobs` and `## Managing scheduled jobs`